compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -300,7 +300,13 @@ pub(crate) fn create_query_frame<
     key: K,
     kind: DepKind,
     name: &'static str,
+    can_call_queries: bool,
 ) -> QueryStackFrame<DepKind> {
+    if !can_call_queries {
+        // Return a minimal query frame if we can't call queries
+        return QueryStackFrame::new(String::new(), None, None, None, kind, None, || Hash64::ZERO);
+    }
```

Comment:
Can this be moved below the `description` formatting? It seems like that shouldn't use any queries.

Reply:
That can and does use queries.